src/components/SelectInput.jsx

```diff
@@ -0,0 +1,53 @@
+import { getMouseEventOptions } from '@testing-library/user-event/dist/utils';
+import React from 'react';
```

Comment:
```suggestion
import React from 'react';
```
- Importação não usada;